app/src/main/java/com/dkanada/chip/views/DisplayView.java

```diff
@@ -34,8 +34,8 @@ public void setDisplay(byte[][] array) {
     public void onDraw(Canvas canvas) {
         super.onDraw(canvas);
 
-        int width = canvas.getWidth();
-        int height = canvas.getHeight();
+        int width = getWidth();
+        int height = getHeight();
```

Comment:
I would leave the canvas keyword here since the drawRect calls are using the canvas as well. I am not sure if this would break anything but since we are drawing to the canvas it might be better to use the height and width of the canvas.